mboard/views.py

```diff
@@ -339,6 +346,9 @@ def post_vote(request):
     user = Session.objects.get(session_key=request.session.session_key)
     rating, _ = Rating.objects.get_or_create(user=user, target=target, board=target.board)
     if vote and target and rating:
+        if rating.vote_time and rating.vote_time > timezone.now() - timedelta(minutes=10):
```

Comment:
```suggestion
        if rating.vote_time and timezone.now() > rating.vote_time + timedelta(minutes=10):
```

The comparison is incorrect, it results in the immediate "expired" status after the first click.